src/main/kotlin/org/assertj/generator/gradle/internal/tasks/AssertionsGeneratorReport.kt

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2023. assertj-generator-gradle-plugin contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.assertj.generator.gradle.internal.tasks
+
+import com.google.common.reflect.TypeToken
+import org.assertj.assertions.generator.AssertionsEntryPointType
+import java.io.File
+import java.util.TreeSet
+
+internal class AssertionsGeneratorReport(

```

Comment:
Comparing locally in IntelliJ via the `Compare to clipboard` is the best way to see the changes here -- there's *very few* that are not language conversion. 

![image](https://user-images.githubusercontent.com/302365/233845992-56965aa3-9f1d-4432-b3b4-f0228c4ffff4.png)
![image](https://user-images.githubusercontent.com/302365/233846023-7d1aef07-cbb9-4e7f-a5b8-ad1e83ea81f9.png)
![image](https://user-images.githubusercontent.com/302365/233846050-af2895c7-0131-45f8-b288-981a75da2d8f.png)
![image](https://user-images.githubusercontent.com/302365/233846071-d6774373-7312-4880-94b2-5b1ad8ffd738.png)